gateway/1.6/modules/ROOT/pages/policies-included-header-injection.adoc

```diff
@@ -9,12 +9,10 @@ endif::[]
 [width="100%", cols="5,15"]
 |===
 >s|Policy name | Header Injection
->s|Summary      | Adds headers to a request or a response
+>s|Summary | Adds headers to a request or a response
```

Comment:
Hey @dhirsch1, I'm not sure why the period is suggested here?

Reply:
Thanks for asking. Because you have a period in another description in this table,  we put them in all descriptions to make them parallel. I don't see this in the Style Guide, so I'll add it.